package.json

```diff
@@ -67,6 +67,11 @@
           ],
           "default": "",
           "description": "Microsoft employee or public contributor"
+        },
+        "docs-build.realTimeValidation.automaticallyEnable": {
+          "type": "boolean",
+          "default": false,
+          "description": "Automatically enable real-time validation"
```

Comment:
With this flag enabled, the real-time validation will be automatically enabled once the opened repository is a valid Docs repository.